test.tf

```diff
@@ -35,6 +35,8 @@ resource "aws_ami" "awsAmiEncrypted" {
     device_name = "dev-name"
     encrypted = "false"
   }
+  
+  
 }
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure bucket policy is enforced with least privileges for all AWS S3 buckets**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
Make sure to note Amazon S3 bucket access control lists (ACLs) that provide read, write, or full-access to 'Everyone'. It is recommended to provide access to S3 buckets to only authenticated users instead.

source: tenablecs-